src/design-system/InputToggle.tsx

```diff
@@ -57,18 +58,19 @@ const ToggleButton = styled.span<{ toggled?: boolean }>`
 
 const InputToggle: React.FC<Props> = (props) => {
   const { toggled, onChange } = props;
+  const inputId = uniqueId("ToggleInput_");
```

Comment:
hm so ... this will generate a new ID on every render, right? I don't think it will break anything but it could cause unnecessary DOM updates. probably not a huge deal but maybe better to store the unique ID once in a state hook and then use that instead?

Reply:
Definitely, that makes sense. I can update this now to use a state hook.